go/libkb/login_session.go

```diff
@@ -81,6 +85,16 @@ func (s *LoginSession) ExistsFor(emailOrUsername string) bool {
 	return true
 }
 
+func (s *LoginSession) NotExpired() bool {
+	now := s.G().GetClock().Now()
+
+	if now.Sub(s.atime) < LoginSessionMemoryTimeout {
+		return true
+	}
```

Comment:
If you're not writing a test to fake time passing (and I don't see one), I'd prefer `time.Since(s.atime) < LoginSessionMemoryTimeout` over using GetClock().